index.html

```diff
@@ -1,69 +1,608 @@
+						</svg>
+					</div>
+
+					<!-- Hero content -->
+					<div class="relative pt-32 pb-10 md:pt-40 md:pb-16">
+
+						<!-- Stats-->
+						<style>
+							@layer utilities {
+								.glow-text {
+									text-shadow: 0 0 10px #ed621d; /* Adjust the shadow spread and color as needed */
+								}
+							}
+						</style>
+
+						<div class="relative pt-16 pb-6 md:pt-24 md:pb-8">
+							<div class="max-w-6xl mx-auto px-4 sm:px-6">
+								<div class="max-w-sm mx-auto text-center">
+									<!-- Single item -->
+									<div class="flex flex-col items-center justify-center md:px-5">
+										<h4 class="font-inter-tight text-15vw font-bold tabular-nums mb-1 text-customOrange glow-text"> <!-- Added glow-text class -->
+											<span x-data="counter(45)" x-text="counterValue">0</span>%
+										</h4>
+										<p class="text-xl text-zinc-300">CEOs that expect their business model to fail within the decade.</p>
```

Comment:
Grammar

Reply:
disagree on this. i know it's not grammatically perfect, but we shouldn't technically be starting sentences with digits that aren't spelled out.

Reply:
I actually didn't know that. Thanks :D
What tripped me up primarily is the sudden relative clause, I think.

So some suggestions:
- An estimated (at the top left or so) 45% (as is) of CEOs expect ... (below as is)
- 45.0% (as is) of CEOs expect .... (otherwise as is) (it seems to be stylistically ok, to use decimals at the beginning of a sentence, but then I'd prefer if it's accurate)
- 45% (as is) [em-dash?] Percentage of CEOs that expect.... (otherwise as is)
- Ignore good style, as it is a stylised text (45% (huge) of CEOs expect... (otherwise as is))